src/components/dialogs/pick-entity-or-conversion.vue

```diff
@@ -265,7 +254,6 @@ export default class PickEntityOrConversionDialog extends CommonMixin {
     // show an URL of creating society NR if Societies NR needs to be released AFTER the way of navigating changes
     if (!this.isSocietyEnabled() && (entity.value === EntityTypes.SO || entity.value === EntityTypes.XSO)) {
       this.showSocietiesInfo = true
-      this.clearEntitySelection()
```

Comment:
...Then I realized that this line was incorrect and the flow worked without it (entity type cd was already cleared when this dialog was shown).